rllib/policy/tests/test_export_checkpoint_and_model.py

```diff
@@ -18,7 +18,7 @@
 # Keep a set of all RLlib algos that support the RLModule API.
 # For these algos we need to disable the RLModule API in the config for the purpose of
 # this test. This test is made for the ModelV2 API which is not the same as RLModule.
-RLMODULE_SUPPORTED_ALGOS = {"PPO"}
+RLMODULE_SUPPORTED_ALGOS = {"APPO", "IMPALA", "PPO"}
```

Comment:
Should we break this up into multiple tests? 

This could become a very long test.

Reply:
:+1: